string shooter_player_id
string hit_color_str
